### ISSUE TITLE
Small arrow on album artwork is not displayed anymore

### ISSUE THREAD

**Issue by ChrisRast** [yianL/atom-spotified#14]
Hi, 
After your last update (0.8.1) I noticed that the small arrow to minify/extend the album artwork is not displayed anymore.
I know this is a cosmetic issue but I just wanted to let you know.
Do you need any other info ?
Cheers,
Chris

**Comment by yianL** [comment]
Thanks for the heads-up. I'll look into it sometime this week.

**Comment by yianL** [comment]
Hi @ChrisRast ,

I couldn't reproduce the issue you're seeing. Perhaps some other extensions you're using has conflicting css classnames. Can you check in the dev console what styles are being applied to `div.toggle` ?

<img width="1440" alt="screen shot 2017-06-25 at 2 50 04 pm" src="https://user-images.githubusercontent.com/741612/27520038-e809c668-59b5-11e7-9eec-a57f3675d8b8.png">

**Comment by ChrisRast** [comment]
Hi,
It seems it's a font-awesome issue.

Here's the styles:
![screen shot 2017-07-03 at 17 41 34](https://user-images.githubusercontent.com/4668052/27799818-f64ecdc4-6016-11e7-9414-349bb13f1062.png)

And the arrow has a height/width of 0x0...
![screen shot 2017-07-03 at 17 42 53](https://user-images.githubusercontent.com/4668052/27799858-230a6422-6017-11e7-92db-5e5c949c517d.png)

I haven't found a rule overriding the declaration so I believe FontAwesome might be the issue?

**Comment by yianL** [comment]
I found a different style overriding the `.fa` class. Turns out I was using font-awesome icons, not knowing it was included by the tool-bar package I installed, not from atom itself. I made a small patch `v0.8.2` to use octicons instead.

<img width="1440" alt="screen shot 2017-07-05 at 11 28 22 am" src="https://user-images.githubusercontent.com/741612/27896133-b9f4f094-61cc-11e7-8eeb-a43316da7653.png">
